### PR TITLE
🐛 136 - test routing fix

### PR DESCRIPTION
**Pull request by anncatton**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -20,7 +20,9 @@ RUN addgroup -S -g $EGO_UI_GID $EGO_UI_USER \
 # defaults
 ENV REACT_APP_EGO_CLIENT_ID=ego
 ENV REACT_APP_API=http://localhost:8081/
-ENV PUBLIC_URL="/"
+ARG REACT_APP_PUBLIC_URL="/"
```
Comment (overturebio) [inline Dockerfile:23]: what was the need for an `ARG`  ?

Comment (blabadi) [inline Dockerfile:23]: what is the need for an ARG ? these are for docker build command

File: Dockerfile [hunk 2]
```diff
@@ -20,7 +20,9 @@ RUN addgroup -S -g $EGO_UI_GID $EGO_UI_USER \
...
+
+ENV REACT_APP_PUBLIC_URL ${REACT_APP_PUBLIC_URL}
```
Comment (blabadi) [inline Dockerfile:25]: this could be `ENV REACT_APP_PUBLIC_URL="/"`

File: Dockerfile (hunk 1 above)
Comment (anncatton) [reply Dockerfile:23]: i had the same setup in dms-ui, i thought that was needed. But can change it back if not!

**Review by blabadi — APPROVED** [review]

**Pull request merged**